src/cache.cpp

```diff
@@ -1100,6 +1100,25 @@ void Cache::fetch_descriptions(RssFeed* feed)
 	run_sql(query, fill_content_callback, feed);
 }
 
+std::string Cache::fetch_description(RssItem* item)
+{
+	std::string in_clause = prepare_query("'%q'", item->guid());
+
+	std::string query = prepare_query(
```

Comment:
Please mark these `const`. (I'm so happy that in Rust, bindings are immutable by default. I won't have to make these nitpicky comments anymore!)

Reply:
Done.
Also marked a few more variables in the same file as `const`.

Reply:
Thanks!